provisioning_virtual_machines_and_hosts/_topics/catalogs_and_services.md

```diff
@@ -372,6 +372,125 @@ Service methods have been split based on purpose.
+
+You need to set the ManageIQ Provisioning service entry point to workflow script. For example, this workflow is run for every provisioned Virtual Machine (VM), and modifies the VM size.
+
+Use the following sections and follow them in order to provision a VM service, create Provisioning Dialog and modify the paramaters.
+
+#### Create a workflow (optional)
+
+**Note**: If you want to use the existing `provision-vm-small-med-large.asl` workflow script without any changes, you can skip this section and proceed to `Add a Workflow Repository` section.
+
+1. Fork `https://github.com/ManageIQ/workflows-examples` repository or create your own repository.
+2. Clone the repo locally by using the clone command:
+
+   ```bash
+   git clone https://github.com/<owner>/workflows-examples
+   ```
+   Where `<owner>` is the owner of the repository.
+
+3. Create a new branch:
+
+   ```bash
+   git checkout -b sm-md-lg
+   ```
+3. Create your own script or edit the existing script that is provided. The script used in this example is `provision-vm-small-med-large.asl`.
+4. Push code changes to your branch.
+
+#### Add a Workflow Repository
+
+Use the following steps to add a Workflow Repository.
+
+1. Click **Automation** > **Embedded Workflows** > **Repositories**.
+2. Click **Configuration** > **Add new Repository**.
+3. Provide the **Name** and **URL** for the repository:
+   - **Name**: Local name to identify this repository. For example, `Example Workflows`.
+   - **Url**: Git repository URL. For example, `https://github.com/ManageIQ/workflows-examples`.
+   - **SCM Branch**: Git repository branch name. For example, leave this field blank if you want to use the existing `provision-vm-small-med-large.asl` script blank or use a branch name like `sm-md-lg` if you have edited or created your own workflow script.
+4. Click **Save**.
+
+#### Create a Provisioning Dialog
+
+Use the following steps to create a Provisioning Dialog.
+
+1. Click **Automation** > **Embedded Automate** > **Customization**.
+2. Click **Service Dialogs** > **All Dialogs**.
+3. Click **Configuration** > **Add a new Dialog**.
+4. Provide a **Name** and **Description** for the dialog:
+   - **Name**: Name used in the Service Catalog. For example, `sm-med-lg dialog`.
+   - **Description**: Description for the dialog. For example, `Dialog choosing vm size`.
+5. Add a text box named **vm_name**.
+6. Add a dropdown named **size**.
+7. Click the edit pencil icon for the **size** dropdown.
+8. Under **Options** > **Entries** enter the following values:
+   - Enter the following for the first row:
+      - For **Name** field, choose **Large**.
+      - For **Value** field, choose **large**.
+   - Enter the following for the second row:
+      - For **Name** field, choose **Medium**.
+      - For **Value** field, choose **medium**.
+   - Enter the following for the third row:
+      - For **Name** field, choose **Small**.
+      - For **Value** field, choose **small**.
```

Comment:
@qasim-ahmed Should the values here being in backticks as opposed to bold - bold feels like the wrong choice, since it's a value to be entered, similar to how above in the "For example,"s we follow those with backticked values.

Reply:
@Fryguy Oh, if you user needs to type these then you are correct. Then the wording should also be type instead of choose.